website/ibm.erb

```diff
@@ -110,11 +110,11 @@
             </li>
           </ul>
         </li>
-        <li<%= sidebar_current("docs-ibm-datasource-dns-zones") %>>
+        <li<%= sidebar_current("docs-ibm-datasource-pdns") %>>
           <a href="#">Private DNS Data Sources</a>
           <ul class="nav nav-visible">
-            <li<%= sidebar_current("docs-ibm-datasource-dns-zones") %>>
```

Comment:
The entry for the dns-zones is overrided please add dns-zones also

Reply:
added